### PR TITLE
Add support to Money field

### PR DESCRIPTION
**Pull request by feliperenan**
Fix https://github.com/plataformatec/simple_form/issues/1579

Since Rails has support to `money` field, I'm adding to Simple Form render this input properly.

### CODE REVIEW

**Comment by feliperenan** [comment]
@frankyston @benoitr @alexanderadam Could some of you test this branch to see if I'm forgetting something? 🙏

File: lib/simple_form/inputs/money_input.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+module SimpleForm
+  module Inputs
+    class MoneyInput < Base
```
Comment (feliperenan) [inline lib/simple_form/inputs/money_input.rb]: Another approach would be handling it inside of `StringInput`. But I created a new input to make it explicit. Does it make sense? 🤔

**Comment by rafaelfranca** [comment]
Since creating a custom input is not hard and I don't think money columns are common enough to require support in the library for them.

**Comment by feliperenan** [comment]
@rafaelfranca Indeed, creating a custom input is pretty straightforward, I didn't think about it 🙈. Do you think we can close this PR then? I'm ok in doing that.

**Comment by rafaelfranca** [comment]
Since the code is here I'm fine with it, but for future it would be better to see how often the type is used to make a decision.

**Comment by feliperenan** [comment]
Since it's not common, I'm going to close this one. If people ask for it we can reopen in the future.

**Pull request closed without merging**